purchase_requisition_line_description/views/purchase_requisition.xml

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" ?>
+<odoo>
+
+    <record id="view_purchase_requisition_form" model="ir.ui.view">
+        <field
+            name="name"
+        >purchase_requisition_line_description.view_purchase_requisition_form</field>
+        <field name="model">purchase.requisition</field>
+        <field
+            name="inherit_id"
+            ref="purchase_requisition.view_purchase_requisition_form"
+        />
+        <field name="arch" type="xml">
+            <field name="product_description_variants" position="attributes">
+                <attribute name="attrs">{}</attribute>
```

Comment:
Is this this required with Odoo 17?

Reply:
@janikvonrotz seems like the module can be deprecated as this functionality is built-in